main.go

```diff
@@ -30,11 +30,14 @@ import (
 	"github.com/google/uuid"
 )
 
-var xdsServerUri = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
-var outputName = flag.String("output", "-", "output file name")
-var gcpProjectNumber = flag.Int64("gcp-project-number", 0,
-	"the gcp project number. If unknown, can be found via 'gcloud projects list'")
-var vpcNetworkName = flag.String("vpc-network-name", "default", "VPC network name")
+var (
+	xdsServerUri     = flag.String("xds-server-uri", "trafficdirector.googleapis.com:443", "override of server uri, for testing")
+	outputName       = flag.String("output", "-", "output file name")
+	gcpProjectNumber = flag.Int64("gcp-project-number", 0,
+		"the gcp project number. If unknown, can be found via 'gcloud projects list'")
+	vpcNetworkName          = flag.String("vpc-network-name", "default", "VPC network name")
+	enableFileWatcherConfig = flag.Bool("enable-file-watcher-config", false, "whether or not to generate file_watcher certificate provider config. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
How did you choose this argument name? I don't see it in the user guide. It seems the name and configuration don't entirely match. It isn't just "file watcher" config. You have to know it is for google_cloud_private_spiffe

Reply:
I just came up with something for the flag name. Didn't realize it was going to be specified in the UG.

As for the name and configuration, I think the UG might need to be updated. What is found here should match the config mentioned in Sanjay's latest doc on file watcher plugin.

Reply:
I don't know why I can't @ mention Sanjay here.

Reply:
You can, it just doesn't auto-complete since he isn't a member of the repository. Calling @sanjaypujare to coordinate this flag name.

Reply:
So only the flag name is being discussed here, right? The generated config should match the following:
```
 "certificate_providers": {
    "google_cloud_private_spiffe": {
        "plugin_name": "file_watcher",
        "config": {
           "certificate_file": "/var/run/gke-spiffe/certs/certificates.pem",
           "private_key_file": "/var/run/gke-spiffe/certs/private_key.pem",
           "ca_certificate_file": "/var/run/gke-spiffe/certs/ca_certificates.pem",
           "refresh_interval": "600s"
        }
    }
  }
```
The UG needs to be updated with some of this latest info. As far as the flag name is concerned I don't know where in the UG it will be mentioned. If people don't like `enable-file-watcher-config` I can suggest `enable-psm-security` which is shorter and also doesn't depend on file-watcher etc.

Reply:
Thank Sanjay.

@ejona86 : Would you prefer the flag name changed to `enable-psm-security`?

Reply:
`--enable-psm-security` sounds good, as it is product-focused. I do think we should probably avoid "enable" though, since it doesn't actually enable any security itself. Maybe `--allow-psm-security` or `--include-psm-security`?

Reply:
I like `--include-psm-security` as it's the most accurate in terms of what the flag is achieving. Let's do that

Reply:
Done. Thanks.